src/debugger/channel/DebugChannel.js

```diff
@@ -95,21 +108,24 @@ export class DebugChannel {
     this._requestReceived = false;
   }
 
-  sendBreakpointAcknowledge(prefix: string, args: BreakpointArguments): void {
+  sendBreakpointAcknowledge(prefix: string, requestID: number, args: BreakpointArguments): void {
```

Comment:
"prefix" does not make any sense here, call it "messageType" or something similar.